### PR TITLE
Fatal error dialog update to reflect open source.

### LINKED ISSUE #623: [Enhancement] Fatal error warning needs updating
### Pitch: what problem are you trying to solve?

Fatal error dialog suggests the error produced has been reported, we have no functionality in actual to send reports anywhere and the email address listed as far as I am aware isn't monitored for the Open Source Project. Example of the warning message below

image

### Describe your ideal solution to this problem

Change the wording to something that fits the project.

### PR DESCRIPTION
**Pull request by rich-howell**
Removed reference to automatic reporting of errors to ActualBudget, also removed the email address as a endpoint for errors to be reported to and replaced it with GitHub issues.

This should resolve #623

### CODE REVIEW

**Comment by j-f1** [comment]
Same as #627, can you update the link?

**Comment by rich-howell** [comment]
> Same as #627, can you update the link?

Updated.

File: packages/desktop-client/src/components/FatalError.js [hunk 1]
```diff
@@ -77,9 +77,9 @@ class FatalError extends React.Component {
         >
           <Text>{msg}</Text>
           <Text>
-            Contact{' '}
-            <a href="mailto:help@actualbudget.com">help@actualbudget.com</a> for
-            support
+            Open an{' '}
+            <a href="https://github.com/actualbudget/actual/issues">issue</a>{' '}
```
Comment (j-f1) [inline packages/desktop-client/src/components/FatalError.js:81]: Can you update this one too?

Comment (rich-howell) [reply packages/desktop-client/src/components/FatalError.js]: Ah yes, that wasn't one of my initial changes so missed it. Updated now.

**Review by j-f1 — APPROVED** [review]

**Pull request merged**